modules/adyoulikeBidAdapter.js

```diff
@@ -2,7 +2,7 @@ import * as utils from '../src/utils.js';
 import { format } from '../src/url.js';
 // import { config } from '../src/config.js';
 import { registerBidder } from '../src/adapters/bidderFactory.js';
-import find from 'core-js/library/fn/array/find.js';
+import find from 'core-js-pure/features/array/find';
```

Comment:
Please add js extension to all imports in all files. More details here https://github.com/prebid/Prebid.js/pull/4876